.travis.yml

```diff
@@ -11,18 +11,14 @@ install:
   - sudo apt-get update
   - sudo apt-get install dotnet-dev-1.0.4 -y
 script: 
-  - echo "========== build all .Net Standard samples ============"
+  - echo "========== build all .Net Core samples ============"
   - dotnet restore
   - dotnet build -c Debug -f netcoreapp1.1 SampleApplications/Samples/NetCoreConsoleClient
   - dotnet build -c Debug -f netcoreapp1.1 SampleApplications/Samples/NetCoreConsoleServer
-  - dotnet build -c Debug -f netcoreapp1.1 SampleApplications/Workshop/Aggregation/ConsoleAggregationServer
-  - dotnet build -c Debug -f netcoreapp1.1 SampleApplications/Workshop/Reference/ConsoleReferenceServer
```

Comment:
Travis issue with dotnet restore. I think building the other two is sufficient anyway.